bosh/opsfiles/volumes.yml

```diff
@@ -0,0 +1,8 @@
+# Required for CredHub because credentials values cannot be an array,
+# so the value here includes the key `export_volumes` that it is replacing
+# in the file path `bosh/manifest.yml` in this repo.
+# https://github.com/cloudfoundry-incubator/credhub-cli/issues/30
+- type: replace
```

Comment:
Why make a new ops file for this? Could we add it to the manifest directly instead?

Reply:
I wanted the minimal amount of work around secrets. And still wanted to keep staging and production unaffected by these changes.

Reply:
IMO the minimal amount of work would be to make a one-line change in the base manifest 😆. We're only going to point dev at this branch, right?

Reply:
Nah. I was going to keep it pointing to master. I didn’t want to add an additional Git resource to the pipeline

Reply:
Considering that this is going to be needed in Staging as well while that CH gets rolled out, I’d rather do it this way and track things we need to revert in a cleanup story.

Reply:
Considering that this is going to be needed in Staging as well while that CH gets rolled out, I’d rather do it this way and track things we need to revert in a cleanup story.